doc/source/developer_guide/dg_dynamics.rst

```diff
@@ -50,9 +50,8 @@ abort if set.  To override the abort, use value 102 for testing.
 Transport
 -----------------
 
-The transport (advection) methods are found in **cicecore/cicedynB/dynamics/**.  Only the incremental
-remapping method is supported at this time, and is set in namelist via the ``advection`` variable.
-Transport can be turned off by setting ``advection = none`` or ``ktransport = -1``.
```

Comment:
It would be good to keep this last sentence in the docs, about how to turn transport off.

Reply:
and maybe reinstate the double quotes around advection

Reply:
This has been updated.